src/games/strategy/debug/DebugUtils.java

```diff
@@ -70,7 +71,7 @@ public static String getMemory() {
     buf.append("Free memory: ").append(runtime.freeMemory() / mb).append("\r\n");
     buf.append("Total memory: ").append(runtime.totalMemory() / mb).append("\r\n");
     buf.append("Max memory: ").append(runtime.maxMemory() / mb).append("\r\n");
-    final int currentMaxSetting = GameRunner.getMaxMemoryFromSystemIniFileInMB(GameRunner.getSystemIni());
+    final int currentMaxSetting = Memory.getMaxMemoryFromSystemIniFileInMB(GameRunner.getSystemIni());
```

Comment:
I don't necessarily disagree. Not sure of a better name at the moment. Regardless, the method was moved out of this class to a new one. To some extent, the name of the method was intentionally not change to allow the diff to be easier.